### PR TITLE
[TR][PAM]PIM-4690: Changed interfaces

### PR DESCRIPTION
**Pull request by solivier**
Interfaces has changed in order to also use the assets categories

| Q                  | A
| ------------------ | ---
| Migration script?  | n
| Behats             | yes
| Specs & Static     | yes
| Changelog updated  | yes

http://core-ci.akeneo.com/job/pim-pr-behat/3042/console

### CODE REVIEW

**Comment by phaseinducer** [comment]
GTM

**Comment by skeleton** [comment]
GTM

**Comment by nidup** [comment]
GTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -177,6 +177,7 @@
 - Add an argument `Pim/Bundle/VersioningBundle/Factory/VersionFactory` in the constructor of `Pim/Bundle/VersioningBundle/Builder/VersionBuilder`
 - Add an argument `Symfony\Component\EventDispatcher\EventDispatcher` in the constructor of `Akeneo\Bundle\StorageUtilsBundle\Doctrine\Common\Remover\BaseRemover`
 - Change constructor of `Pim\Bundle\BaseConnectorBundle\Reader\ORM\CategoryReader`, argument is now a `Pim\Component\Classification\Repository\CategoryRepositoryInterface`
+- Replace interface `Pim\Bundle\CatalogBundle\Model\CategoryInterface` by `Pim\Component\Classification\Model\CategoryInterface`
```
Comment (jjanvier) [inline CHANGELOG.md]: remove that, you didn't bc break something

File: spec/Pim/Bundle/TransformBundle/Normalizer/Flat/CategoryNormalizerSpec.php [hunk 2]
```diff
@@ -3,7 +3,7 @@
 namespace spec\Pim\Bundle\TransformBundle\Normalizer\Flat;
 
 use PhpSpec\ObjectBehavior;
-use Pim\Bundle\CatalogBundle\Model\CategoryInterface;
+use Pim\Component\Classification\Model\CategoryInterface;
```
Comment (jjanvier) [inline spec/Pim/Bundle/TransformBundle/Normalizer/Flat/CategoryNormalizerSpec.php]: it seems you have forgotten the real class

**Pull request merged**